features/networking/service.feature

```diff
@@ -441,16 +441,18 @@ Feature: Service related networking scenarios
   Scenario: Taint node with too small MTU value
     Given the default interface on nodes is stored in the :default_interface clipboard
     And the node's MTU value is stored in the :mtu_actual clipboard
+    And the node's active nmcli connection is stored in the :nmcli_active_con_uuid clipboard
     And evaluation of `node.name` is stored in the :subject_node clipboard
     And I run commands on the host:
-      | systemctl stop NetworkManager                        |
-      | ip link set mtu 1300 dev <%= cb.default_interface %> |
+      | nmcli con modify  "<%= cb.nmcli_active_con_uuid %>"  ethernet.mtu  1300  |
+      | nmcli dev reapply <%= cb.default_interface %> |
```

Comment:
@asood-rh test only runs till this line and then it fails and there is a bug open for the same.  https://bugzilla.redhat.com/show_bug.cgi?id=1930446